be/src/exec/pipeline/pipeline_driver.h

```diff
@@ -158,6 +158,7 @@ class PipelineDriver {
         for (auto& op : _operators) {
             _operator_stages[op->get_id()] = OperatorStage::INIT;
         }
+        snprintf(_driver_name, sizeof(_driver_name), "Driver(node=%d/id=%d)", _source_node_id, _driver_id);
```

Comment:
Why not a `std::string`? `snprintf` is so old school.

Reply:
Fixed.